pom.xml

```diff
@@ -67,7 +67,7 @@
   <properties>
     <changelist>999999-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.319.1</jenkins.version>
+    <jenkins.version>2.340</jenkins.version>
```

Comment:
FWIW intentionally not updating the bom artifact ID right now to reduce conflicts with dependabot

Reply:
WDYM? Should it not be set to `weekly`? Or at least `2.332.x`?